app/main.py

```diff
@@ -1,17 +1,11 @@
-def count_occurrences(phrase: str, letter: str) -> int:
-    """
-    Implement count_occurrences function:
+from cmath import phase
 
-    It takes a phrase and a letter and calculates the number of times
-    the letter appears in the phrase. The function is case insensitive.
+from pyflakes.checker import counter
 
-    count_occurrences("letter", "t") == 2
-    count_occurrences("abc", "a") == 1
-    count_occurrences("abc", "d") == 0
-    count_occurrences("ABC", "a") == 1
 
-    :param phrase: phrase to count in it
-    :param letter: letter to find occurrences of it
-    :return: count occurrences of letter in phrase
-    """
-    # write your code here
+def count_occurrences(phrase: str, letter: str) -> int:
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+             counter += 1
+    return counter
```

Comment:
not bad solution but try to make it in one line. You can use already existed method that counts appearance of char in string, for example :)

Reply:
still not fixed